brianchandotcom/liferay-portal
# LPS-179883 Copy permissions on the new page when a Page is created with Copy Page

[Link to the story](https://issues.liferay.com/browse/LPS-177031)

### Motivation
Copy permissions on the new page when a Page is created with Copy Page

### Proposed Solution
Extract copy page to the api with copy permissions
Add new dropdown item contextual to use it in the item list of actions.

### Steps to Verify:
Open the Site Builder
Create a new Blanck Page called Page 1
Publish Page 1
Click on One Page Options Menu -> Permissions
Remove all the permissions for Guest
Click on One Page Options Menu -> Copy Page with Permissions
Name new page as Page 2
Verify that new permissions are similar to the copied page

### Notes
Integration test included.


cc.: @Mikellorza @ruben-pulido
